com.unity.netcode.adapter.utp/Runtime/NetworkMetricsPipelineStage.cs

```diff
@@ -0,0 +1,53 @@
+#if MULTIPLAYER_TOOLS
+using AOT;
+using Unity.Burst;
+using Unity.Collections.LowLevel.Unsafe;
+using Unity.Networking.Transport;
+using UnityEngine;
+
+namespace Unity.Netcode
+{
+    [BurstCompile]
+    public unsafe struct NetworkMetricsPipelineStage : INetworkPipelineStage
+    {
+        static TransportFunctionPointer<NetworkPipelineStage.ReceiveDelegate> ReceiveFunction = new TransportFunctionPointer<NetworkPipelineStage.ReceiveDelegate>(Receive);
+        static TransportFunctionPointer<NetworkPipelineStage.SendDelegate> SendFunction = new TransportFunctionPointer<NetworkPipelineStage.SendDelegate>(Send);
+        static TransportFunctionPointer<NetworkPipelineStage.InitializeConnectionDelegate> InitializeConnectionFunction = new TransportFunctionPointer<NetworkPipelineStage.InitializeConnectionDelegate>(InitializeConnection);
+
+        public NetworkPipelineStage StaticInitialize(byte* staticInstanceBuffer, int staticInstanceBufferLength, NetworkSettings settings)
+        {
+            return new NetworkPipelineStage(ReceiveFunction,
+                                            SendFunction,
+                                            InitializeConnectionFunction,
+                                            ReceiveCapacity: UnsafeUtility.SizeOf<int>(),
+                                            SendCapacity: UnsafeUtility.SizeOf<int>(),
+                                            HeaderCapacity: UnsafeUtility.SizeOf<ushort>(),
```

Comment:
Should set all of these to 0 until needed.

Reply:
Will do.